libs/fixtures/src/index.ts

```diff
@@ -11,10 +11,6 @@ export {
   electionDefinition as electionGeneralDefinition,
   election as electionGeneral,
 } from './data/electionGeneral/election.json';
-export {
-  electionDefinition as electionMarkScanDiagnosticDefinition,
-  election as electionMarkScanDiagnostic,
-} from './data/electionGeneral/election.json';
```

Comment:
This should have been `./data/electionMarkScanDiagnostic/...` but this fixes the issue by just cloning `electionGeneral` to `mark-scan/backend`